proto/src/determined/api/v1/api.proto

```diff
@@ -2382,4 +2394,49 @@ service Determined {
       tags: "Internal"
     };
   }
-}
+
+  // Get task config
+  rpc GetGenericTaskConfig(GetGenericTaskConfigRequest)
+      returns (GetGenericTaskConfigResponse) {
+    option (google.api.http) = {
+      get: "/api/v1/tasks/{task_id}/config"
+    };
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+      tags: "Internal"
+    };
+  }
+
+  // Kill generic task
+  rpc KillGenericTask(KillGenericTaskRequest)
+      returns (KillGenericTaskResponse) {
+    option (google.api.http) = {
+      post: "/api/v1/tasks/{task_id}/kill"
+      body: "*"
+    };
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+      tags: "Internal"
+    };
+  }
+
+  // Pause generic task
+  rpc PauseGenericTask(PauseGenericTaskRequest)
+      returns (PauseGenericTaskResponse) {
+    option (google.api.http) = {
+      post: "/api/v1/tasks/{task_id}/pause"
+    };
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+      tags: "Internal"
+    };
+  }
+
+  // Resume generic task
+  rpc ResumeGenericTask(ResumeGenericTaskRequest)
+      returns (ResumeGenericTaskResponse) {
+    option (google.api.http) = {
+      post: "/api/v1/tasks/{task_id}/resume"
+    };
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+      tags: "Internal"
+    };
+  }
+}
```

Comment:
Can you configure your editor to save the final newline in files?  That's an old unix standard, and it helps other developers not create spurious git diffs when they edit the same file later.